docs/recipes/structuring-reducers/UpdatingNormalizedData.md

```diff
@@ -148,6 +148,7 @@ import comments from "./commentsReducer";
 import dotProp from "dot-prop-immutable";
 import {combineReducers} from "redux";
 import reduceReducers from "reduce-reducers";
+import {createReducer} from 'redux-orm'
```

Comment:
Wouldn't be coming from `redux-orm`.  I think in this case it was a purely notional example based on our "Reducing Boilerplate" docs page, since I wrote this back in 2016.

Reply:
Ok then Let me remove it